LITRON/iocBoot/iocLITRON-IOC-01/st-common.cmd

```diff
@@ -0,0 +1,44 @@
+epicsEnvSet "STREAM_PROTOCOL_PATH" "$(LITRON)/data"
+epicsEnvSet "DEVICE" "L0"
+epicsEnvSet "NUM_PORT" "L1"
+
+##ISIS## Run IOC initialisation 
+< $(IOCSTARTUP)/init.cmd
+
+## Device simulation mode IP configuration
+$(IFDEVSIM) drvAsynIPPortConfigure("$(DEVICE)", "localhost:$(EMULATOR_PORT=57677)")
+
+## For recsim:
+$(IFRECSIM) drvAsynSerialPortConfigure("$(DEVICE)", "$(PORT=NUL)", 0, 1, 0, 0)
```

Comment:
$(DEVICE) is created for devsim and recsim, but not the real system. I think you don't actually use the string port $(DEVICE) but we should be consistent.   We should either remove $(DEVICE) if we are sure we never need the string port, or we should change `DEVICE` to `STRING_PORT` and add something like
```
drvAsynIPPortConfigure("$(STRING_PORT)", "$(IPADDR):64008 TCP")
asynOctetConnect("STRINGINIT","$(STRING_PORT)")
asynOctetWrite("STRINGINIT" "*IDN? ")
```